JavaScriptSPA/authConfig.js

```diff
@@ -0,0 +1,27 @@
+
+// Config object to be passed to Msal on creation.
+// For a full list of msal.js configuration parameters, 
+// visit https://azuread.github.io/microsoft-authentication-library-for-js/docs/msal/modules/_configuration_.html
+const msalConfig = {
+  auth: {
+    clientId: "e760cab2-b9a1-4c0d-86fb-ff7084abd902",
+    authority: "https://fabrikamb2c.b2clogin.com/fabrikamb2c.onmicrosoft.com/b2c_1_susi",
+    validateAuthority: false
+  },
+  cache: {
+    cacheLocation: "localStorage", // This configures where your cache will be stored
+    storeAuthStateInCookie: false // Set this to "true" to save cache in cookies to address trusted zones limitations in IE (see: https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/Known-issues-on-IE-and-Edge-Browser)
+  }
+};
+
+// Add here scopes for id token to be used at the MS Identity Platform endpoint
+// For a full list of available authentication parameters, 
+// visit https://azuread.github.io/microsoft-authentication-library-for-js/docs/msal/modules/_authenticationparameters_.html
+const loginRequest = {
+  scopes: ["openid", "profile"],
+};
+
+// Add here scopes for access token to be used at the API endpoints.
+const tokenRequest = {
+  scopes: ["https://fabrikamb2c.onmicrosoft.com/helloapi/demo.read"]
```

Comment:
Does this const still need to be defined here if it's already _apiConfig.js_?

From _apiConfig.js_ L2-4:

```JavaScript
// The current application coordinates were pre-registered in a B2C tenant.
const apiConfig = {
  b2cScopes: ["https://fabrikamb2c.onmicrosoft.com/helloapi/demo.read"],
  webApi: "https://fabrikamb2chello.azurewebsites.net/hello"
```

Reply:
technically it isn't. I wanted to make `authConfig.js` file self-contained, and use `apiConfig.js` file only to clarify and make explicit what are the relation of these strings to the API.

Reply:
Do you think this is more clear?:

```
const tokenRequest = {
  scopes: apiConfig.b2cScopes 
}
```

Reply:
The references in authPopup and authRedirect would need to be updated, of course.

Reply:
Actually, that wouldn't work since `getTokenPopup` is probably expecting a `scopes` property on the passed-in object.

Reply:
how about?

```
const tokenRequest = {
  scopes: apiConfig.b2cScopes  //e.g. ["https://fabrikamb2c.onmicrosoft.com/helloapi/demo.read"]
}
```

Reply:
I like it. The developer needs then to update the API scope URI in only one const in the project, but they're also made aware of the type it's expecting.

Reply:
alright pushing the changes

Reply:
@sameerag Are you OK with this change? It avoids having to specify the same const value twice, while still leaving `tokenRequest` available in `authConfig.js`.

https://github.com/Azure-Samples/active-directory-b2c-javascript-msal-singlepageapp/pull/52/files#r398934715

Reply:
Can we get this going today @sameerag @mmacy ? Hopefully it will help with a user's issue : ))

Reply:
Super close - I just need someone to walk through all the docs as if they've never seen them before to make sure all the steps function as expected and result in a successful experience. I've done so, but I'd like at least one other person to help verify.

Here are the staging links:

* [Quickstart](https://review.docs.microsoft.com/en-us/azure/active-directory-b2c/quickstart-single-page-app?branch=pr-en-us-105885)
* [Tutorial - SPA](https://review.docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-single-page-app?branch=pr-en-us-105885&tabs=app-reg-preview) (part 1)
* [Tutorial - SPA calls API](https://review.docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-single-page-app-webapi?branch=pr-en-us-105885&tabs=app-reg-preview) (part 2)

Corresponding PR in docs: https://github.com/MicrosoftDocs/azure-docs-pr/pull/105885

Reply:
> @sameerag Are you OK with this change? It avoids having to specify the same const value twice, while still leaving `tokenRequest` available in _authConfig.js_.

Yes @mmacy. 
``` javascript
const tokenRequest = {
  scopes: apiConfig.b2cScopes 
} 
```
looks like proper approach, as we do not want to pass `scopes` as an individual object.